app/src/main/java/edu/wisc/ece/pinpoint/fragments/ProfilePageFragment.java

```diff
@@ -50,19 +47,22 @@ public void onTabSelected(TabLayout.Tab tab) {
 
             @Override
             public void onTabUnselected(TabLayout.Tab tab) {
-                // Mandatory override meethod left intentionally blank because we will not implement onTabUnselected
+                // Mandatory override method left intentionally blank because we will not
+                // implement onTabUnselected
             }
 
             @Override
             public void onTabReselected(TabLayout.Tab tab) {
-                // Mandatory override meethod left intentionally blank because we will not implement onTabReselected
+                // Mandatory override method left intentionally blank because we will not
+                // implement onTabReselected
             }
         });
 
         viewPager.registerOnPageChangeCallback(new ViewPager2.OnPageChangeCallback() {
             @Override
             public void onPageSelected(int position) {
                 super.onPageSelected(position);
+                //noinspection ConstantConditions
```

Comment:
What is this for?

Reply:
This is to remove the warning for the potential NullPointerException on the line below. We can decide to just keep the warning and remove the comment or vice versa. Open to opinions.

Reply:
I think having the comment to remove the warning is better.